internal/conf/nacos_test.go

```diff
@@ -57,6 +57,20 @@ func TestNacosValidator(t *testing.T) {
 				fmt.Errorf("Timeout.Connect: Must be greater than or equal to 1\nWeight: Must be greater than or equal to 1"),
 			},
 		},
+		{
+			caseDesc: "Test User",
+			givePath: "user.yaml",
+			wantValidateErr: []error{
+				fmt.Errorf("User: Does not match pattern '^[a-zA-Z0-9-_.]*$'"),
```

Comment:
So neither username nor password can contain `@` ?

Reply:
`https://username:password@console.nacos.io:8858`
the `@` is used to split user&password and host, so the origin username and password don't contain the `@` character, so i just delete it from the validation, if it is needed, we could add it later